### DISCUSSION TITLE
load certs from /etc/ipsec.d/certs so that listcerts will show the certs

### DISCUSSION THREAD

**Discussion by usmcfiredog** [strongswan/strongswan#471] [Q&A]
Hello is there a way for strongswan to read the directory /etc/ipsec.d/certs? I want to be able to do that so that I can run IPsec listcerts in order to see all of the certs that I have created on my machine. I know there is the ipsec reread commands but none of them read the certs folder.

**Comment by tobiasbrunner** [comment]
No, the legacy starter/stroke config backend only loads certificates referenced in `left|rightcert`. If you want to load trusted certificates without referencing them in a config use [swanctl](https://wiki.strongswan.org/projects/strongswan/wiki/swanctl), whose `--load-creds` command loads certificates in the `swanctl/x509` directory automatically.

**Reply by usmcfiredog** [reply]
ok but my problem is I do not have a command swanctl on my system how do I go about enabling this?

**Reply by tobiasbrunner** [reply]
You didn't say what system that is, so no idea (but if you installed strongSwan from binary packages, there might be one that provides swanctl/vici). But maybe you should first ask yourself whether you actually need this, i.e. what your goal is with loading plain certificates like that.
